recipes/bwa-mem2/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "2.0" %}
+
+package:
+  name: bwa-mem2
+  version: {{ version }}
+
+source:
+  - url: https://github.com/bwa-mem2/bwa-mem2/releases/download/v{{ version }}/Source_code_including_submodule.tar.gz
+    sha256: 23a8aa9d295ed1e042aec255157313b0ddf45ae2271a9b878b5e05a4c1026d38
+    patches:
+      - Makefile.patch
+      - 771c7692aeb857d5b9241a08bbd68f2c6be01811.patch
+
+build:
+  number: 0
+  # Compilation fails on OSX:
+  #   ext/safestringlib/include/safe_mem_lib.h:100:16: error: conflicting types for 'memset_s'
+  #   extern errno_t memset_s(void *dest, rsize_t dmax, uint8_t value);
+  #   /Applications/Xcode-9.4.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.sdk/usr/include/string.h:145:9: note: previous declaration is here
+  #   errno_t memset_s(void *__s, rsize_t __smax, int __c, rsize_t __n) __OSX_AVAILABLE_STARTING(__MAC_10_9, __IPHONE_7_0);
+  # Cause: https://github.com/intel/safestringlib/issues/14
+  skip: True  # [osx]
```

Comment:
If someone wants to try to patch things to make it compile under macOS, feel free to do so! (Preferably in a subsequent PR ;).